### PR TITLE
Fix swagger

### PR DESCRIPTION
**Pull request by darioAnongba**
Fix swagger

### CODE REVIEW

**Review by 6l20 — APPROVED** [review]

File: src/stores/entities/annotations.go [hunk 1]
```diff
@@ -1,9 +1,9 @@
 package entities
 
 type Annotation struct {
-	AWSKeyID             string `json:"aws_key_id,omitempty"`
-	AWSCustomKeyStoreID  string `json:"aws_custom_key_store_id,omitempty"`
-	AWSCloudHsmClusterID string `json:"aws_cloud_hsm_cluster_id,omitempty"`
-	AWSAccountID         string `json:"aws_account_id,omitempty"`
-	AWSArn               string `json:"aws_arn,omitempty"`
+	AWSKeyID             string `json:"AWSKeyID,omitempty"`
```
Comment (dolanor) [inline src/stores/entities/annotations.go:4]: If the json name is the same of the Go struct field, you don't have to specify it. You can just put `json:",omitempty"`

**Comment by dolanor** [comment]
This doesn't generate the swagger files for me:
```
```

**Review by dolanor — CHANGES_REQUESTED** [review]
This doesn't generate the swagger files for me:
```
2021/09/20 17:13:37 Generate swagger docs....
2021/09/20 17:13:37 Generate general API Info, search dir:./src
2021/09/20 17:13:37 Generating types.SetSecretRequest
2021/09/20 17:13:37 Generating types.SecretResponse
2021/09/20 17:13:37 Generating handlers.ErrorResponse
2021/09/20 17:13:37 Generating handlers.PageResponse
2021/09/20 17:13:37 Generating handlers.PagePagingResponse
2021/09/20 17:13:37 Generating types.CreateEthAccountRequest
2021/09/20 17:13:37 Generating types.EthAccountResponse
2021/09/20 17:13:37 Generating types.ImportEthAccountRequest
2021/09/20 17:13:37 Generating types.UpdateEthAccountRequest
2021/09/20 17:13:37 Generating types.SignMessageRequest
2021/09/20 17:13:37 Generating types.SignTypedDataRequest
2021/09/20 17:13:37 Generating types.DomainSeparator
2021/09/20 17:13:37 Generating types.Type
2021/09/20 17:13:37 Generating types.SignETHTransactionRequest
2021/09/20 17:13:37 Generating types.SignEEATransactionRequest
2021/09/20 17:13:37 Generating types.SignQuorumPrivateTransactionRequest
2021/09/20 17:13:37 Generating types.AliasRequest
2021/09/20 17:13:37 Generating types.AliasResponse
2021/09/20 17:13:37 Generating types.Alias
2021/09/20 17:13:37 Generating types.VerifyKeySignatureRequest
2021/09/20 17:13:37 ParseComment error in file src/stores/api/handlers/utils.go :example value Cjix/fS3WdqKGKabagBNYwcClan5aImoFpnjSF0cqJs= can't convert to integer err: strconv.Atoi: parsing "Cjix/fS3WdqKGKabagBNYwcClan5aImoFpnjSF0cqJs=": invalid syntax
make: *** [Makefile:138: gen-swagger] Error 1

```
It seems it doesn't like to have this content stored inside a `[]byte`.
```
./src/stores/api/types/keys.go
35:	PublicKey        []byte `json:"publicKey" validate:"required" example:"Cjix/fS3WdqKGKabagBNYwcClan5aImoFpnjSF0cqJs="`
```

**Comment by darioAnongba** [comment]
Good catch! I fixed the issue

File: src/stores/entities/annotations.go (hunk 1 above)
Comment (darioAnongba) [reply src/stores/entities/annotations.go:4]: This may be personal but IMO it's better to always name

**Review by dolanor — APPROVED** [review]

**Pull request merged**